util/log/file.go

```diff
@@ -46,7 +56,8 @@ var logDir *string
 var logDirs []string
 
 // logFileRE matches log files to avoid exposing non-log files accidentally.
-var logFileRE = regexp.MustCompile(`log\.(INFO|WARNING|ERROR)\.`)
+// and it splits the details of the filename into groups for easy parsing.
+var logFileRE = regexp.MustCompile(`([^\.]+)\.([^\.]+)\.([^\.]+)\.log\.(ERROR|WARNING|INFO)\.([^\.]+)\.(\d+)`)
```

Comment:
Should include an example of the format in this comment, or in the parse method below.